src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -0,0 +1,119 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/*!
+ * \file
+ * Declares functions to retrieve spectral quantities associated with a
+ * particular choice of basis functions and quadrature method.
+ */
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+
+class Matrix;
```

Comment:
cond this

Reply:
Sorry what do you mean?

Reply:
Doxygen can sometimes mess up when presented with forward declarations so we do this to ignore them in the docs:
```
/// \cond
class Matrix; 
/// \endcond 
```